File: web-example/src/api/resp/ServiceResponse.ts

/**
 * 服务响应对象
**/

export interface  ServiceResponse<PageInfo<T>> {

        /**
            *响应数据消息
            *在java中的类型为：String
        **/
        message?: string;
        /**
            *响应数据code
            *在java中的类型为：Integer
        **/
        code?: number;
        /**
            *响应数据
        **/
        data?: T;
}

File: examples/swagger-3/src/test/resources/asserts/java/spring_http/src/com/wuxp/codegen/swagger3/resp/ServiceResponse.java
package com.wuxp.codegen.swagger3.resp;

import lombok.Data;
import lombok.experimental.Accessors;
import jakarta.validation.constraints.*;
import com.fasterxml.jackson.annotation.JsonFormat;

@Data
@Accessors(chain = true)
public class  ServiceResponse<T> {

            /**
                *字段在java中的类型为：String
            **/
         String message;

            /**
                *字段在java中的类型为：Integer
            **/
         Integer code;

         T data;

}
